package.json

```diff
@@ -38,24 +30,18 @@
     "ip-address": "^6.2.0",
     "jsonfile": "^2.4.0",
     "jsonwebtoken": "^8.3.0",
-    "later": "^1.2.0",
     "lodash": "^4.17.15",
     "minimatch": "3.0.4",
     "minimist": "*",
     "mkdirp": "^0.5.1",
-    "mobile-detect": "https://registry.npmjs.org/mobile-detect/-/mobile-detect-1.3.3.tgz",
     "moment": "^2.15.0",
-    "morgan": "^1.9.1",
     "mustache-express": "^1.2.5",
     "network": "git+https://git@github.com/melvinto/network.git",
     "node-df": "^0.1.4",
     "node-forge": "^0.7.1",
     "node-persist": "0.0.8",
     "node-ssdp": "^4.0.0",
-    "passport": "*",
```

Comment:
in use

Reply:
This doesn't appear to be in use outside the `api` directory, which has its own `package.json` file:

https://github.com/firewalla/firewalla/blob/4550b0756ce72d253290d437c5f7d54a8d213005/api/package.json#L16